### PR TITLE
XM2C-43 Merge docker-share into docker-alfresco

### PR DESCRIPTION
**Pull request by hechmi-dammak-xenit**

### CODE REVIEW

**Review by thijslemmens — CHANGES_REQUESTED** [review]
No proper title, no description. Please fix

File: tomcat-base/src/shared/main/java/eu/xenit/alfresco/tomcat/embedded/alfresco/AlfrescoTomcatCustomizer.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package eu.xenit.alfresco.tomcat.embedded.alfresco;
```
Comment (hechmi-dammak-xenit) [inline tomcat-base/src/shared/main/java/eu/xenit/alfresco/tomcat/embedded/alfresco/AlfrescoTomcatCustomizer.java:1]: I think that this class should be under the tomcat package too same for shareTomcatCustomizer

File: README.md [hunk 2]
```diff
@@ -1,98 +1,163 @@
...
 * [`alfresco-repository-community`] = community Alfresco images
+* [`docker.io/xenit/share-enterprise`] = enterprise Share images
+* [`docker.io/xenit/share-community`] = community Share images
 
-## Supported Tags
+## Supported Tags for Alfresco
 
 * [`:7.2.0`, `:7.2`, `:7`] = major, minor, revision
 
+## Supported Tags for Share
```
Comment (thijslemmens) [inline README.md:24]: This section is not up-to-date

File: README.md [hunk 3]
```diff
@@ -1,98 +1,163 @@
...
-This project builds Alfresco-specific docker images used by Xenit, starting with Alfresco 4.2.
+This project builds Alfresco-specific docker images used by Xenit, starting with Alfresco 6.1.
 
-## Images created 
+## Images created
 
 * [`alfresco-skeleton`] = skeleton common for all images per major version; includes java, tomcat, init script, keystore
 * [`alfresco-repository-enterprise`] = enterprise Alfresco images
 * [`alfresco-repository-community`] = community Alfresco images
+* [`docker.io/xenit/share-enterprise`] = enterprise Share images
```
Comment (thijslemmens) [inline README.md:17]: enterprise images should never go to docker.io

File: README.md [hunk 4]
```diff
@@ -1,98 +1,163 @@
...
-> [tools](https://xenit.eu/open-source), [services](https://xenit.eu/alfresco) and 
+> a strong team around Alfresco. If you'd like to learn more about our
+> [tools](https://xenit.eu/open-source), [services](https://xenit.eu/alfresco) and
 > [products](https://xenit.eu/alfresco-products), please visit our [website](https://xenit.eu).
 
-This project builds Alfresco-specific docker images used by Xenit, starting with Alfresco 4.2.
+This project builds Alfresco-specific docker images used by Xenit, starting with Alfresco 6.1.
 
-## Images created 
+## Images created
```
Comment (thijslemmens) [inline README.md:12]: please check and correct the full image url's. Now it is inconsistent between share and repository

**Review by thijslemmens — CHANGES_REQUESTED** [review]
Almost there. The README needs some attention. I tested it to work.

File: README.md [hunk 5]
```diff
@@ -11,21 +11,15 @@ This project builds Alfresco-specific docker images used by Xenit, starting with
...
 ## Images created
 
-* [`alfresco-skeleton`] = skeleton common for all images per major version; includes java, tomcat, init script, keystore
-* [`alfresco-repository-enterprise`] = enterprise Alfresco images
-* [`alfresco-repository-community`] = community Alfresco images
-* [`docker.io/xenit/share-enterprise`] = enterprise Share images
-* [`docker.io/xenit/share-community`] = community Share images
-
-## Supported Tags for Alfresco
+* [`docker.io/xenit/tomcat`] = Xenit specific tomcat images
```
Comment (thijslemmens) [inline README.md:14]: Didn't know we publish this one too, but apparently we do

File: README.md [hunk 6]
```diff
@@ -11,21 +11,15 @@ This project builds Alfresco-specific docker images used by Xenit, starting with
...
+* [`docker.io/xenit/alfresco-skeleton`] = skeleton common for all images per major version; includes java, tomcat, init script, keystore
+* [`docker.xenit.eu/private/alfresco-enterprise/alfresco-repository-enterprise`] = enterprise Alfresco images
+* [`docker.io/xenit/alfresco-repository-community`] = community Alfresco images
+* [`docker.io/xenit/docker.io/xenit/share-community`] = community Share images
```
Comment (thijslemmens) [inline README.md:18]: please recheck. `docker.io` twice, and if I read your code, I'm expecting `alfresco-share-community`

**Review by thijslemmens — CHANGES_REQUESTED** [review]

**Review by thijslemmens — APPROVED** [review]

**Pull request merged**